oak/proto/permissions.proto

```diff
@@ -0,0 +1,41 @@
+//
+// Copyright 2020 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.permissions;
+
+// A PermissionsConfiguration provides a declarative description of the features that are permitted
+// for an Oak application.
+//
+// The permissions are applied dynamically at runtime, when creating the Nodes.
+message PermissionsConfiguration {
+  // Whether to enable the use of a GrpcServerNode.
+  bool allow_grpc_server_nodes = 1;
+
+  // Whether to enable the use of an HttpServerNode.
+  bool allow_http_server_nodes = 2;
+
+  // Whether to allow application logging. If true, allows creating a LogNode for the application.
+  bool allow_logging = 3;
```

Comment:
Since there is runtime logging and application logging, and this is really about the log pseudo node, why not just keep a consistent terminology and make the bools refer to actual node types? it seems simpler?

Reply:
Good point. I changed the name. 

`allow_introspection` and `allowed_grpc_egress_addresses` do not directly refer to a node type. So, I did not change them. But let me know if you can think of better names :)